infra/watchdog/batcher/README.md

```diff
@@ -0,0 +1,36 @@
+# Batcher Watchdog
+
+The Batcher Watchdog checks a prometheus metric and restart the batcher as needed
```

Comment:
A bit more detail would be appreciated. Which metric is checked? How often is it checked? What does "as needed" mean? I know the answers because we  sit close to each other, but someone in the future will not know this.